script.js

```diff
@@ -140,4 +140,13 @@ function projectPopup(position) {
   popup.classList.add('popup');
   portfolios.appendChild(popup);
 }
-
+const projectnBtn = document.querySelectorAll('.see-project');
+projectnBtn.forEach((btn, ind) => {
+  btn.addEventListener('click', () => {
+    projectPopup(ind);
+    const popup = document.querySelector('.popup');
+    document.querySelector('.close-icon').addEventListener('click', () => {
+      portfolios.removeChild(popup);
+    });
+  });
+});
```

Comment:
- [ ] Good implementation of popup, but you still have something to fix on your design. The popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) :upside_down_face: :upside_down_face:. In your case when I open the popup to see the project content I need to scroll down, I think it should be better when the user clicks the `see project` button can see all the popup content without scrolling down :grin: :grin: 

Your design
![Screenshot from 2022-07-21 19-22-09](https://user-images.githubusercontent.com/74501523/180275778-b9af4a2c-d5d0-4f6d-87d0-236f24965545.png)

Figma design
![Screenshot from 2022-07-21 19-05-21](https://user-images.githubusercontent.com/74501523/180274477-b6d4a193-331f-4e1e-8c3b-841c6cf42337.png)

- [ ] Please kindly make sure your image is visible, so then the user can see very well the project's picture :grin: :grin: :upside_down_face:

Reply:
- [ ] [REQUIRED]  Good job for implementing the changes requested by the previous reviewer, your popup looks very nice :clap:  but I think you can improve your popup by increasing the size of your popup. Please make sure that you have the dimensions as on the Figma template and increase your popup background color  `background: rgb(193 199 208 / 50%).`  and   ` backdrop-filter: blur(3px)`  to cover the all website details behind your popup :+1: 

- [ ] [REQUIRED] The previous reviewer requested changes to your project pictures to make it more visible and more clear for use please apply the changes :smile: 

#### Your project
![Screenshot from 2022-07-22 00-06-18](https://user-images.githubusercontent.com/6615394/180336321-dcf6d4a6-277f-4801-bd3e-847402cc617c.png)

#### Figma
![Screenshot from 2022-07-22 00-24-07](https://user-images.githubusercontent.com/6615394/180336427-e2076076-9a86-4901-b34d-b19078747aaa.png)